ts/components/conversation/Quote.tsx

```diff
@@ -126,235 +105,257 @@ export class Quote extends React.Component<Props, State> {
-            convoId={convoId}
-            text={text}
-            disableLinks={true}
-            i18n={i18n}
-          />
-        </div>
-      );
-    }
-
-    if (!attachment) {
-      return null;
-    }
+  return (
+    <div className="module-quote__generic-file">
+      <div className="module-quote__generic-file__icon" />
+      <div
+        className={classNames(
+          'module-quote__generic-file__text',
+          isIncoming ? 'module-quote__generic-file__text--incoming' : null
+        )}
+      >
+        {fileName}
+      </div>
+    </div>
+  );
+};
 
-    const { contentType, isVoiceMessage } = attachment;
-
-    const typeLabel = getTypeLabel({ i18n, contentType, isVoiceMessage });
-    if (typeLabel) {
-      return (
-        <div
-          className={classNames(
-            'module-quote__primary__type-label',
-            isIncoming ? 'module-quote__primary__type-label--incoming' : null
-          )}
-        >
-          {typeLabel}
-        </div>
-      );
-    }
+export const QuoteIconContainer = (props: any) => {
+  const { attachment, i18n, imageBroken, handleImageErrorBound } = props;
 
+  if (!attachment) {
     return null;
   }
 
-  public renderClose() {
-    const { onClose } = this.props;
+  const { contentType, thumbnail } = attachment;
+  const objectUrl = getObjectUrl(thumbnail);
 
-    if (!onClose) {
-      return null;
-    }
+  if (GoogleChrome.isVideoTypeSupported(contentType)) {
+    return objectUrl && !imageBroken ? (
+      <QuoteImage url={objectUrl} i18n={i18n} icon={'play'} />
+    ) : (
+      <QuoteIcon icon="movie" />
+    );
+  }
+  if (GoogleChrome.isImageTypeSupported(contentType)) {
+    return objectUrl && !imageBroken ? (
+      <QuoteImage
+        url={objectUrl}
+        i18n={i18n}
+        contentType={contentType}
+        handleImageErrorBound={handleImageErrorBound}
+      />
+    ) : (
+      <QuoteIcon icon="image" />
+    );
+  }
+  if (MIME.isAudio(contentType)) {
+    return <QuoteIcon icon="microphone" />;
+  }
+  return null;
+};
 
-    // We don't want the overall click handler for the quote to fire, so we stop
-    //   propagation before handing control to the caller's callback.
-    const onClick = (e: any): void => {
-      e.stopPropagation();
-      onClose();
-    };
+export const QuoteText = (props: any) => {
+  const { i18n, text, attachment, isIncoming, conversationType, convoId } = props;
 
-    // We need the container to give us the flexibility to implement the iOS design.
+  if (text) {
     return (
-      <div className="module-quote__close-container">
-        <div className="module-quote__close-button" role="button" onClick={onClick} />
+      <div
+        dir="auto"
+        className={classNames(
+          'module-quote__primary__text',
+          isIncoming ? 'module-quote__primary__text--incoming' : null
+        )}
+      >
+        <MessageBody
+          isGroup={conversationType === 'group'}
```

Comment:
try replacing the 'private' and the 'group' string comparison with the `ConversationTypeEnum.GROUP`. when you see a string like that